internal/monitors/kubernetes/apiserver/metadata.yaml

```diff
@@ -0,0 +1,1443 @@
+monitors:
+- monitorType: kubernetes-apiserver
+  doc: |
+      ```
```

Comment:
Needs a bit more description on what this monitor is useful for.  Maybe include a link to that article you mentioned.